services/vision/vision.go

```diff
@@ -108,6 +108,25 @@ func init() {
 //	if len(objects) > 0 {
 //			logger.Info(objects[0])
 //	}
+//
+// CaptureAllFromCamera example:
+//
+//	// The data to capture and return from the camera
+//	captOpts := viscapture.CaptureOptions{
+//	ReturnImage:           true,
+//			ReturnClassifications: true,
+//			ReturnDetections:      true,
+//			ReturnObject:          true,
+//	}
```

Comment:
Just a note: currently in the implementation for the CaptureAllFromCamera collector, it tries to return everything it can so setting the ReturnImage: true, ReturnClassifications: true, etc. doesn't actually do anything. Not sure if/when this may be prioritized - @tahiyasalam would have a better answer!

Reply:
This is not a priority, so I would just leave captOpts empty in the example